pkg/ipmi/blobs/blob_transfer.go

```diff
@@ -0,0 +1,429 @@
+// Copyright 2020 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// OpenBMC IPMI Blob Protocol commands
```

Comment:
Turn this into a package comment by removing the empty line below and structuring it like a usual Go package comment, "Package blobs ...blabla"